src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -1776,7 +1776,9 @@ def list_resources(cmd, resource_group_name=None,
     odata_filter = _list_resources_odata_filter_builder(resource_group_name,
                                                         resource_provider_namespace,
                                                         resource_type, name, tag, location)
-    resources = rcf.resources.list(filter=odata_filter)
+
+    expand = "createdTime,changedTime,provisioningState"
+    resources = rcf.resources.list(filter=odata_filter, expand=expand)
```

Comment:
is there a way to get expand value not hard code? In addition, could we expose the parameter to customer to let customer choose if they want to expand or not?

Reply:
Because these fields are all returned before, but they are no value. Therefore, making them optional fields can only reduce the output data, such usage scenarios may be rare.